### PR TITLE
Updated fake message body format to object

### PR DESCRIPTION
**Pull request by jillcary**
# Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [x] The code follows the code style and conventions of this project.
- [x] The change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.

# Description of the change

The fake messags returned by GET /v1/messages now
include a body that is in JSON format for a fake device
that includes:
- temperature
- t_unit (units used for temp)
- humidity
- latitutde
- longitude

# Motivation for the change

Updated the message format to be more realistic for the UI to call and display device data

### CODE REVIEW

File: app/com/microsoft/azure/iotsolutions/devicetelemetry/services/Messages.java [hunk 1]
```diff
@@ -25,10 +27,49 @@ public Messages() {
...
+        sampleMessage = new SampleMessage(
+            "74",
+            "F",
+            "41",
+            "47.642272",
+            "-122.103374");
+
+        jsonBody = Json.toJson(sampleMessage);
+
+        sampleMessages.add(new MessageServiceModel("Elevator1",
```
Comment (ppathan) [inline app/com/microsoft/azure/iotsolutions/devicetelemetry/services/Messages.java]: nit: match the format to other lines. Either all the parameters on new line or none.

File: app/com/microsoft/azure/iotsolutions/devicetelemetry/services/SampleMessage.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+*/
+public class SampleMessage {
+
+    public String temperature = null;
+    public String t_unit = null;
+    public String humidity = null;
+    public String latitude = null;
+    public String longitude = null;
+
+    @Inject
```
Comment (ppathan) [inline app/com/microsoft/azure/iotsolutions/devicetelemetry/services/SampleMessage.java]: Is this required?

File: app/com/microsoft/azure/iotsolutions/devicetelemetry/services/SampleMessage.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+package com.microsoft.azure.iotsolutions.devicetelemetry.services;
+
+import com.google.inject.Inject;
+
+/*
+* This class is used to return a fake JSON body for the messages class.
+*/
+public class SampleMessage {
```
Comment (ppathan) [inline app/com/microsoft/azure/iotsolutions/devicetelemetry/services/SampleMessage.java]: Don't we need a Message object class for actual message, wondering why only for sample?

File: app/com/microsoft/azure/iotsolutions/devicetelemetry/services/Messages.java [hunk 4]
```diff
@@ -25,10 +27,49 @@ public Messages() {
      */
     private ArrayList<MessageServiceModel> getSampleMessages() {
         ArrayList<MessageServiceModel> sampleMessages = new ArrayList<>();
+        JsonNode jsonBody = null;
```
Comment (dluc) [inline app/com/microsoft/azure/iotsolutions/devicetelemetry/services/Messages.java]: as discussed offline, replace JsonNode across the board with "Object"

**Pull request merged**